.gitignore

```diff
@@ -1,13 +1,10 @@
 *.iml
 .gradle
 /local.properties
-/.idea/caches
-/.idea/libraries
-/.idea/modules.xml
-/.idea/workspace.xml
-/.idea/navEditor.xml
-/.idea/assetWizardSettings.xml
+/.idea/
```

Comment:
Please extend the ignore rules to cover the mose commonly used editors in our company: you have commited the `*.swp` file from Vim.

Reply:
Yup